app/src/main/java/com/hedvig/app/feature/referrals/ui/PieChartView.kt

```diff
@@ -0,0 +1,71 @@
+package com.hedvig.app.feature.referrals.ui
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.RectF
+import android.util.AttributeSet
+import android.view.View
+import androidx.annotation.ColorInt
+import androidx.dynamicanimation.animation.FloatValueHolder
+import androidx.dynamicanimation.animation.SpringAnimation
+import androidx.dynamicanimation.animation.SpringForce
+
+class PieChartView @JvmOverloads constructor(
+    context: Context,
+    attributeSet: AttributeSet? = null,
+    defStyle: Int = 0
+) : View(context, attributeSet, defStyle) {
+
+    private val circle = RectF(0f, 0f, 1000f, 1000f)
+    private val colorStash = HashMap<Int, Paint>()
+
+    var segments: List<PieChartSegment> = emptyList()
+        set(value) {
+            field = value
+            invalidate()
+        }
+
+    fun reveal(finalSegments: List<PieChartSegment>) {
+        SpringAnimation(FloatValueHolder())
+            .apply {
+                spring = SpringForce().apply {
+                    dampingRatio = 0.65f
```

Comment:
const?

Reply:
I agree with this principle - however, the `PieChartView` snuck in a bit early in this PR and this code will likely not exist by the next PR. Is it ok if we defer these fixes until that PR?

Reply:
I'm deferring more fixes until a later PR.